javers-core/src/main/java/org/javers/common/reflection/JaversFieldFactory.java

```diff
@@ -34,7 +41,20 @@ public JaversFieldFactory(Class methodSource) {
         return fields;
     }
 
-    private JaversField createJField(Field rawField, TypeResolvingContext context){
+    public JaversField getField(String fieldName) {
```

Comment:
hmm, to się raczej powinno nazywac getDeclaredFiels() bo pomijasz logike dziedziczenia która jest obsłużona w getAllFields(), czy to jest zamierzone?